scrapers/covid-19/getVariants.js

```diff
@@ -0,0 +1,72 @@
+const axios = require("axios");
+const logger = require("../../utils/logger");
+const csvUtils = require("../../utils/csvUtils");
+
+const PATH =
+    "https://opendata.ecdc.europa.eu/covid19/virusvariant/csv/data.csv";
```

Comment:
Keep this in mind when writing the Swagger docs
![image](https://user-images.githubusercontent.com/4297028/149625541-4081f521-c583-4671-b6f5-2d5d2177b260.png)